### ISSUE TITLE
Why is 16k frequency not supported

### ISSUE THREAD

**Issue by Prinfoxce** [gindemit/unity-wrapper-vorbis#7]
I want to use a 16k frequency for recording, but this library restricts the frequency between 44100 and 192000. Otherwise, it returns ERROR_INVALID_FREQUENCY_PARAMETER. Why is this, and can 16k be supported?

**Comment by gindemit** [comment]
Hello! Thanks for your interest in the plugin. I just changed the minimal frequency parameter check from 44k to 8k. Can you please download the binary files below and put those into your Unity Assets folder where the VorbisPlugin is located. 
Please let me know if it works for you.

[VorbisPluginLinuxArtifacts.zip](https://github.com/user-attachments/files/17894778/VorbisPluginLinuxArtifacts.zip)
[VorbisPluginWindowsArtifacts.zip](https://github.com/user-attachments/files/17894781/VorbisPluginWindowsArtifacts.zip)
[VorbisPluginMacOSArtifacts (1).zip](https://github.com/user-attachments/files/17894779/VorbisPluginMacOSArtifacts.1.zip)
[VorbisPluginIOSArtifacts (2).zip](https://github.com/user-attachments/files/17894780/VorbisPluginIOSArtifacts.2.zip)

**Comment by Prinfoxce** [comment]
Thank you! This solved my problem, and I was able to test it with the 16k frequency working fine. Also, will you be releasing a new version on the release page? I noticed that not all Android ABIs are included in the LinuxArtifacts, such as armeabi-v7a.

**Comment by gindemit** [comment]
Hi @tagleweb, sorry for the late reply. You can now install the plugin from Unity package manager.

Please use this URL to get the latest version

```
https://github.com/gindemit/unity-vorbis.git?path=/unity/Assets/VorbisPlugin/#0.4.1
```

**Comment by hippogamesunity** [comment]
Hello! I'm sorry for the delay. I've checked both ways to import the asset (copy files / with manifest), in Unity 2022 and 6.
All they still show the same warning:

<img width="579" height="1261" alt="Image" src="https://github.com/user-attachments/assets/a8576c70-d02b-499a-84ef-2c8abacc871a" />

**Comment by gindemit** [comment]
Hi @hippogamesunity, there is a bug in Unity - it shows the warning even if the libraries are aligned.
You can use the cli from Android SDK to make sure those are aligned. Or upload the build to Google Play.

Here is a thread from my other plugin where the users had the same problem:
https://github.com/gindemit/unity-rlottie/issues/48#issuecomment-3304039003

**Comment by hippogamesunity** [comment]
Yes, I was able to upload my build to Google Play with no warnings. Thanks
for your time!

пт, 16 янв. 2026 г., 14:38 Gindemit Konstantin ***@***.***>:
